test/WebJobs.Script.Tests/DependencyTests.cs

```diff
@@ -32,7 +32,7 @@ public class DependencyTests
         private readonly DependencyContextJsonReader _reader = new DependencyContextJsonReader();
         private readonly IEnumerable<string> _rids = DependencyHelper.GetRuntimeFallbacks();
 
-        [Fact]
+        [Fact(Skip = "Will update later")]
```

Comment:
👍🏼 

Can we please create an issue to track this and the full, thorough review of runtimeassemblies.txt?

Reply:
Filed https://github.com/Azure/azure-functions-host/issues/7349